popmon/visualization/histogram_section.py

```diff
@@ -140,16 +140,33 @@ def transform(self, data_obj: dict, sections: Optional[list] = None):
             ]
             plots = parallel(_plot_histograms, args)
 
+            layouts = []
+            plot_types = []
             # filter out potential empty plots
             plots = [e for e in plots if len(e["plot"])]
             plots = sorted(plots, key=lambda plot: plot["name"])
-
+            if len(plots) > 0:
+                layouts.append(plots[0]["layout"])
```

Comment:
Now that zip is moved from the template to the logic, we can see that if the layouts are associated with the plot type, then a dictionary/mapping might be a more appropriate data structure.